src/sestest/sestest.csproj

```diff
@@ -2,7 +2,7 @@
 
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFramework>netcoreapp1.1</TargetFramework>
+    <TargetFramework>netcoreapp2.0</TargetFramework>
     <Company>Microsoft</Company>
     <Authors>Azure Batch</Authors>
     <Product>Azure Batch Software Entitlement Service SDK</Product>
```

Comment:
Again here (can't seem to aim my comment correctly on this file!)

Reply:
Removed.